opencti-platform/opencti-graphql/src/manager/telemetryManager.ts

```diff
@@ -125,8 +132,11 @@ const fetchTelemetryData = async (manager: TelemetryMeterManager) => {
     manager.setInstancesCount(clusterInfo.info.instances_number);
     // endregion
     // region Users information
-    const users = await getEntitiesListFromCache(context, TELEMETRY_MANAGER_USER, ENTITY_TYPE_USER);
+    const users = await getEntitiesListFromCache(context, TELEMETRY_MANAGER_USER, ENTITY_TYPE_USER) as AuthUser[];
     manager.setUsersCount(users.length);
+    // FIXME is is active users or connected users ?
+    // const activeUser = users.filter((user: AuthUser) => user.account_status === ACCOUNT_STATUS_ACTIVE);
+    // manager.setActiveUsersCount(activeUser.length);
```

Comment:
@richard-julien do we put back the active user gauge ? And if yes, should it be based on current active user sessions or on active users overall ?

Reply:
For me we can fully remove this one as its not really a good activity metric